var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -34,7 +34,9 @@ class LibjpegTurbo(AutotoolsPackage):
     homepage = "http://libjpeg-turbo.virtualgl.org"
     url      = "http://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-1.3.1.tar.gz"
 
-    version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257')
+    version('1.5.1', '55deb139b0cac3c8200b75d485fc13f3')
+    version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257', preferred=True)
```

Comment:
Why is 1.5.0 preferred?  We don't normally do that when adding a new release, unless there's a serious problem with the new one.